components/i2c/scd30/definition.json

```diff
@@ -0,0 +1,21 @@
+{
+  "displayName": "SCD30",
+  "i2cAddresses": [ "0x61"],
+  "subcomponents": [
+    {
+      "displayName": "Temperature Sensor",
+      "sensorType": "ambient-temp",
+      "defaultPeriod": 900
+    },
+    {
+      "displayName": "Humidity Sensor",
+      "sensorType": "humidity",
+      "defaultPeriod": 900
+    },
+    {
+        "displayName": "CO2 Sensor",
+        "sensorType": "co2",
+        "defaultPeriod": 900
+      }
```

Comment:
superfluous indentation here

Reply:
Fixed indentation in https://github.com/adafruit/Wippersnapper_Components/pull/13/commits/e05a526c13c8f20271a957fc55746069776bfefc